examples/deployment-manager/autogen/multivm/README.md

```diff
@@ -39,6 +39,10 @@ variables in `configurations.yaml`
 mpdev cfg set redis/ projectId $PROJECT_ID
 mpdev cfg set redis/ image $IMAGE
 mpdev cfg set redis/ defaultReplicas 4
+
+# Optionally set deployment package zip output path.
+# Can also be in GCS (e.g. gs://<BUCKET>/<OBJECT>/<GENERATION>)
```

Comment:
Remove `/<GENENERATION>` as generations are automatically generated (they are timestamps). Also, generation is the `#1234567` part of the object name.